lib/bank_accounts/money_market_account.rb

```diff
@@ -0,0 +1,141 @@
+module BankAccounts
+  class MoneyMarketAccount < Account
+    ##--------------------------------------------------------------------------
+    # CONSTANTS
+
+
+    # limits
+    MINIMUM_BALANCE = 10_000
+    TRANSACTION_LIMIT = 6
+
+    # fees
+    LOW_BALANCE_FEE = 100
+
+
+    ##--------------------------------------------------------------------------
+    # INSTANCE METHODS
+
+
+    attr_reader :transactions
+
+
+    def initialize(id, initial_balance)
+      if (initial_balance >= MINIMUM_BALANCE)
+        @transactions = 0
+
+        super(id, initial_balance)
+
+        return true
+
+      else
+        raise ArgumentError.new("You cannot create a money market account with less than a $#{ MINIMUM_BALANCE } initial deposit.")
+      end
+    end
```

Comment:
Same comments here -- you can eliminate the if block by moving the `raise` to the top, and you don't need to return anything.